packages/cli/src/utils/token/loadTokenUtils.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { consola } from "consola";
+import * as fs from "node:fs";
+import * as path from "node:path";
+
+const TOKEN_ENV_VARS: string[] = ["FOUNDRY_TOKEN", "FOUNDRY_SDK_AUTH_TOKEN"];
+
+/**
+ * Loads a JWT Auth Token from an argument, a file, or environment variable.
+ * @param token The token as a string.
+ * @param tokenFile The path to the token file.
+ * @returns The token as a string.
+ * @throws An error if no token is found.
+ */
+export function loadToken(token?: string, tokenFile?: string): string {
+  if (token) {
+    checkIsValidToken(token);
+    consola.debug(`Using token from --token argument`);
+    return token;
+  }
+
+  if (tokenFile) {
+    const loadedToken = loadTokenFile(tokenFile);
+    checkIsValidToken(loadedToken);
+    consola.debug(`Using token from --tokenFile argument`);
```

Comment:
Are we going to log the file path used as well? I know this one is in the CLI args but may be still helpful to have the paths used always in the debug stdout for verbose mode